src/components/modals/providers-bottom-sheet.tsx

```diff
@@ -39,20 +46,50 @@ export function ProvidersBottomSheet({
     };
   }, [onPressProvider, eventSuffix]);
 
+  const visibleProviders = useMemo(() => {
+    if (!searchProviderValue) {
+      return providers;
+    }
+
+    return providers.filter(provider =>
+      provider.name
```

Comment:
Add more conditions by these fields for flexible search:

	•	coinName
	•	ethChainId
	•	denom